about/policy/license-disclaimer/index.md

```diff
@@ -34,4 +34,8 @@ Please add attribution with a logo from our [media resources]({% link about/medi
 ## Disclaimer
 {: .text-left}
 
-This product is for informational purposes and may not have been prepared for, or be suitable for legal, engineering, or surveying purposes. Users of this information should review or consult the primary data and information sources to ascertain the usability of the information. AGRC provides these data in good faith and shall in no event be liable for any incorrect results, any lost profits and special, indirect or consequential damages to any party, arising out of or in connection with the use or the inability to use the data hereon or the services provided. AGRC shall not be held liable for any third party’s interpretation of data provided by AGRC. AGRC provides these data and services as a public service. AGRC reserves the right to change or revise published data and/or these services at any time.
+These data, including (but not limited to) geographic data, tabular data, or analytical data, are provided "as is" without any express or implied warranties.
```

Comment:
@jacobdadams should "express" be "expressed or implied warranties"?

Reply:
I think "express" is pretty common verbiage in this scenario.

Reply:
I think it's "express". If I understand the UCC link correctly, there are "express" warranties and "implied" warranties—"express" and "implied" are adjectives, not verbs, so we don't change the tense.